lib/travis/build/bash/travis_getaddrinfo.bash

```diff
@@ -0,0 +1,14 @@
+travis_getaddrinfo() {
+  local nodename="${1}"
+
+  if [[ ! "${nodename}" ]]; then
+    return
+  fi
+
+  # shellcheck disable=SC1010
+  rvm "$(travis_internal_ruby)" --fuzzy do ruby -rsocket <<EORUBY
```

Comment:
Do we need rvm here? Seems straight-up `ruby` should work.

Reply:
I'm happy to use `ruby` alone if that's an option!

Reply:
Let's do that.